include/RAJA/policy/cuda/MemUtils_CUDA.hpp

```diff
@@ -299,184 +299,207 @@ cudaDeviceProp& device_prop()
 
 struct CudaFixedMaxBlocksData
 {
-  int multiProcessorCount;
-  int maxThreadsPerMultiProcessor;
+  int device_sm_per_device;
+  int device_max_threads_per_sm;
 };
 
 RAJA_INLINE
-size_t cuda_max_blocks(size_t block_size)
+CudaFixedMaxBlocksData cuda_max_blocks()
```

Comment:
I think it would be helpful for others to understand if a comment block was provided above each struct and method in this file containing a concise explanation of its purpose